src/g42-main.cpp

```diff
@@ -302,6 +303,7 @@ int main(int argc, const char **argv) {
 	the_program->take_args(argt_exe , argt); // takes again args, with removed special early args
 	the_program->startup_console_first();
 	the_program->startup_version();
+	my_cap::drop_privileges_on_startup(); // [SECURITY] drop unneeded privileges (more will be dropped later)
```

Comment:
We need at least working console, and good to show program version at very beginning;
Documented this.